### PR TITLE
IDE-975 Gray out items in Find Repository panel for local repositories

### PR DESCRIPTION
**Pull request by dehilsterlexis**
…e Repo

Signed-off-by: David de Hilster <david.dehilster@lexisnexis.com>

### CODE REVIEW

File: eclide/RepositoryFilterView.cpp [hunk 1]
```diff
@@ -463,6 +463,20 @@ void CRepositoryFilterView::DoRefreshState()
     {
         m_search.EnableWindow(true);
     }
+
+    CString serverIP = GetIConfig(QUERYBUILDER_CFG)->Get(GLOBAL_SERVER_IP);
+    if (!serverIP.GetLength()) {
```
Comment (GordonSmith) [inline eclide/RepositoryFilterView.cpp:468]: I am surprised this works as you will still have a Global Server IP while in local repo mode?
`IsLocalRepositoryEnabled` would be a better test.

**Review by GordonSmith — CHANGES_REQUESTED** [review]

File: eclide/RepositoryFilterView.cpp (hunk 1 above)
Comment (dehilsterlexis) [reply eclide/RepositoryFilterView.cpp:468]: IsLocalRepositoryEnabled checks if g_EnableCompiler is true. But the grayed out functionalities are for only where there is a server with an IP address.

Comment (dehilsterlexis) [reply eclide/RepositoryFilterView.cpp]: I stand corrected!

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by GordonSmith** [comment]
@dehilsterlexis can you tweak the commit message, so the title fits with guide lines (and fits in the readme)

**Comment by dehilsterlexis** [comment]
@GordonSmith changed

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**